core/src/test/scala/org/http4s/blaze/util/ExecutionSpec.scala

```diff
@@ -89,5 +88,21 @@ class ExecutionSpec extends Specification {
 
       i must be equalTo(iterations)
     }
+
+    "not catch fatal exceptions" in {
```

Comment:
This is the one that was reported.  Testing the others would be tedious, but welcome if someone is so ambitious.